event.go

```diff
@@ -407,21 +395,22 @@ func (e *Event) Err(err error) *Event {
 	if e == nil {
 		return e
 	}
+
 	if e.stack && ErrorStackMarshaler != nil {
 		switch m := ErrorStackMarshaler(err).(type) {
 		case nil:
+			// do nothing
 		case LogObjectMarshaler:
-			e.Object(ErrorStackFieldName, m)
+			e = e.Object(ErrorStackFieldName, m)
```

Comment:
Fix the lost `*Event` returned from these methods, when the old code continued at line 414, it was running against the original `*Event`... technically, this doesn't break anything because event is actually modifying the context's buffers, but this is the only place I noticed where we're not following the fluent conventions... could bite if there is ever contextual/state information added to the `Event`